### PR TITLE
[13.0][MIG] sale_procurement_amendment

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on :

- [x] #1564

### CODE REVIEW

**Review by Cedric-Pigeon — APPROVED** [review]

File: setup/sale_order_line_serial_unique/.eggs/README.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```
Comment (sbidoul) [inline setup/sale_order_line_serial_unique/.eggs/README.txt]: Do not commit these .eggs directory, please. I recommend putting `.eggs/` in your global personal .gitignore (along with your IDE files etc).

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Comment by rafaelbn** [comment]
Hello @rousseldenis the dependencies are solved so you can rebase and move it forward 😄 thank you!

**Comment by rousseldenis** [comment]
> Hello @rousseldenis the dependencies are solved so you can rebase and move it forward smile thank you!

Hi @rafaelbn ! Great to see you back!

I'll do it. If you can help me closing/merging old PR's, pinging authors it could be great on this repo (245 PR's is too much)

**Comment by rafaelbn** [comment]
> Hi @rafaelbn ! Great to see you back!

🤟 

> I'll do it. If you can help me closing/merging old PR's, pinging authors it could be great on this repo (245 PR's is too much)

I've just taged around 20, I expected that ocabot closed the old ones 😆 . Don't worry now there are only 243!! 2 less!

Let's go! Gogogo!

**Comment by rafaelbn** [comment]
Rebuilding travis to check it

**Comment by rousseldenis** [comment]
> Rebuilding travis to check it

I'll try to check the test that fail today

**Comment by rousseldenis** [comment]
@rafaelbn This one is ready (and 14 is already merged)

**Comment by rafaelbn** [comment]
> I'll do it. If you can help me closing/merging old PR's, pinging authors it could be great on this repo (245 PR's is too much)

236 😄

**Pull request merged**